HTML + zdjecia

vldmrtss opened the pull request:
Mam nadzieje ze dobrze zrobilem HTML. Dodalem zdjecia dla calej sekcji (1x i 2x) dla mob, tab, desk. + dodalem sprite svg(hero.svg). Chyba wszystko:)

vldmrtss committed:
HTML + zdjecia

UrszulaBaranowska reviewed (commented):
1. Cyferki nie będą h2>- wrzucamy je albo w span albo p
2. Musisz podpiąć hero.html do index.html w odpowiednim miejscu :) 
Mam wątpliwości co do kropek - bo w htmlu nie dzielimy na mobile czy desktop - ale faktycznie w tablecie i mobile tego środkowego elementu brakuje czyli jest tak jakby hidden. Zapytam Mentora.

vldmrtss committed:
pare zmian html

vldmrtss committed:
podpięcie hero do index

UrszulaBaranowska reviewed (commented):
czekamy na pozostałe kropki :)

vldmrtss committed:
feauture/hero

dodalem zdjecia dla wszystkich wersji, podlaczylem plik css, ustawione clasy do tagów

UrszulaBaranowska reviewed (approved):
Wygląda dobrze :) brakuje cssa na background bo sama fota bez pozycjonownia to malo ale zapewne dopiszesz dalej :)
